e2e/stack.go

```diff
@@ -266,3 +283,51 @@ func (s *Stack) runMonomer(ctx context.Context, env *environment.Env, genesisTim
 	}
 	return nil
 }
+
+type mockAccountRetriever struct {
```

Comment:
optional - the `mockAccountRetriever`/`mockAccount` structs and methods could be pulled out in a separate file for readability

Reply:
Good idea. 9d1eafb